src/controller/audio-stream-controller.js

```diff
@@ -273,7 +277,7 @@ class AudioStreamController extends BaseStreamController {
         }
         if (frag) {
           // logger.log('      loading frag ' + i +',pos/bufEnd:' + pos.toFixed(3) + '/' + bufferEnd.toFixed(3));
-          if (frag.encrypted) {
+          if (frag.encrypted && !this.hls.config.emeEnabled) {
```

Comment:
@realeyes-matthew So is this block of code just for AES-128 and we're skipping if DRM?

Reply:
Exactly, sometimes it would detect that a frag was encrypted but we don't want to use the AES-decypter if the content is EME-protected